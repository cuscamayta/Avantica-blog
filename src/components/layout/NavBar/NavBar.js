import React, { Component } from 'react';

class NavBar extends Component {
    render() {
        return (
            <nav className="navbar navbar-expand-lg bg-dark navbar-dark fixed-top" id="mainNav">
                <div className="container">
                    <a className="navbar-brand" href="#/posts">NewTech Blog</a>
                    <button className="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
                        Menu
                <i className="fa fa-bars"></i>
                    </button>
                    <div className="collapse navbar-collapse" id="navbarResponsive">
                        <ul className="navbar-nav ml-auto">
                            <li className="nav-item">
                                <a className="nav-link" href="#/posts">Posts</a>
                            </li>
                            <li className="nav-item">
                                <a className="nav-link" href="#/about">About</a>
                            </li>
                        </ul>
                    </div>
                </div>
            </nav>
        )
    }
}

export default NavBar;