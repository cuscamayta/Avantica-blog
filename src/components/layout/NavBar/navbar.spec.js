import React from 'react';
import NavBar from './NavBar';
import renderer from 'react-test-renderer';

describe('navbar', () => {
  it('renders navbar component correctly', () => {
    const tree = renderer.create(
      <NavBar />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
