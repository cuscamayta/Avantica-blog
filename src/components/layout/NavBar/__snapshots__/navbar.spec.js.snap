// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar renders navbar component correctly 1`] = `
<nav
  className="navbar navbar-expand-lg bg-dark navbar-dark fixed-top"
  id="mainNav"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="#/posts"
    >
      NewTech Blog
    </a>
    <button
      aria-controls="navbarResponsive"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler navbar-toggler-right"
      data-target="#navbarResponsive"
      data-toggle="collapse"
      type="button"
    >
      Menu
      <i
        className="fa fa-bars"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarResponsive"
    >
      <ul
        className="navbar-nav ml-auto"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="#/posts"
          >
            Posts
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="#/about"
          >
            About
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
